app/git/utils.py

```diff
@@ -590,11 +591,40 @@ def get_interested_actions(github_url, username, email=''):
     return actions_by_interested_party
 
 
-def get_user(user, sub_path=''):
+def get_user(user, sub_path='', auth=_AUTH):
     """Get the github user details."""
     user = user.replace('@', '')
     url = f'https://api.github.com/users/{user}{sub_path}'
```

Comment:
The request need to be https://api.github.com/user/orgs?access_token=XXX
otherwise is not going to give you the private orgs.
By default when you create or add an Org to you profile it gets private not public and is not even visible in `https://api.github.com/users/:username/orgs?access_token=XXX` you need to access with scope `https://api.github.com/user/orgs?access_token=XXX`

Reply:
How I see my profile
![image](https://user-images.githubusercontent.com/1358093/65775074-da078700-e115-11e9-9ff7-c357daa8b378.png)
How is showed 
![image](https://user-images.githubusercontent.com/1358093/65775125-f277a180-e115-11e9-9258-9bd05188f257.png)

So just with the scoped endpoint you will able to see it 
Similar with   with https://api.github.com/orgs/gitcoinco/members?access_token=XX Will give you everyone private and public but in that case without access_token just the publics ones (I believe the access_token need to be from an admin)

Reply:
`get_user` works the same as it was previously, my addition lets the auth token be set in the calling function as an override.

organizations are fetched inside the `sync_profile` function and set to the profile object there.

I can make the changes to the sync step to account for fetching private organizations, and look into the oAuth scope for that.

Reply:
yes but note that the scope calls are diff to be able to get the private info, even with token
`users/:username/orgs`  need to be `/user/orgs`

Reply:
i updated the query to check the `user` scope vs the `users` scope